### PR TITLE
issue #1066 - remove unused strings

### PR DESCRIPTION
**Pull request by alicoding**
To test this patch just run `grunt validate` this should not fail on any of the unused strings at all.

Please test on any of the webmaker page to see if there is any error or missing strings (displaying the key name instead of the actual string).

### CODE REVIEW

**Comment by alicoding** [comment]
@secretrobotron can you r? this please? :)

File: gtf-ignored-keys.json [hunk 1]
```diff
@@ -3,5 +3,7 @@
   "eoy-name-7", "eoy-quote-8", "eoy-name-8", "advanced", "beginner", "intermediate", "Webpages with Thimble", "Are you sure you want to delete this make?",
   "Confirm", "Created by", "created", "Updated", "updated", "unpublished", "Resources", "Resource Templates",
   "Remixes with X-Ray Goggles", "NoEventsFound", "Multimedia with Popcorn", "make", "Make", "Details", "Explore",
-  "DetailsShareTwitterMsg", "guide", "Info"
+  "DetailsShareTwitterMsg", "guide", "Info", "Did you know Webmaker is also available in", "Webmaker", "StandardFooterTranslate",
+  "StandardConnectingCollaboratingPoint1", "StandardConnectingCollaboratingPoint2", "StandardConnectingCollaboratingPoint3",
+  "StandardFooterButton", "StandardMainDesc1", "StandardMainDesc2", "StandardMainDesc3", "StandardMainHeader"
```
Comment (secretrobotron) [inline gtf-ignored-keys.json]: This is a pretty confusing situation. We need to mention things like `StandardMainDesc1` here, even though it's translated by an angular filter? Can we just prevent `public/views/literacy.html` from being checked or something?

File: locale/en_US/appmaker.json [hunk 2]
```diff
@@ -48,7 +52,7 @@
     "description": "'Imagine' blurb content at the bottom of the Appmaker landing page"
   },
   "Make an app for your...": {
-    "message": "Make an app for your <span id=\"app-audience\" data-show=\"{{showAudienceTitle}}\">{{audiences[currentSlideIndex].title | i18n}}</span>",
+    "message": "Make an app for your <span id=\"app-audience\" data-show=\"{{showAudienceTitle}}\">{{audiences[currentSlideIndex].title | i18n }}</span>",
```
Comment (secretrobotron) [inline locale/en_US/appmaker.json]: Don't you want to add a space to the left side as well? :P

Comment (alicoding) [inline locale/en_US/appmaker.json]: I should :P but this won't be validate because they are variable :(

File: gtf-ignored-keys.json (hunk 1 above)
Comment (alicoding) [inline gtf-ignored-keys.json]: yeah I think it is better to just not sure `literacy.html` at all...

**Comment by secretrobotron** [comment]
r+

**Pull request merged**